test/sql/domain_field_types.sql

```diff
@@ -0,0 +1,198 @@
+begin;
+
+  CREATE DOMAIN domain_int AS int;
+  CREATE DOMAIN domain_smallint AS smallint;
+  CREATE DOMAIN domain_bigint AS bigint;
+  CREATE DOMAIN domain_numeric AS numeric;
+  CREATE DOMAIN domain_real AS real;
+  CREATE DOMAIN domain_double_precision AS double precision;
+  CREATE DOMAIN domain_boolean AS boolean;
+  CREATE DOMAIN domain_text AS text;
+  CREATE DOMAIN domain_character AS character;
+  CREATE DOMAIN domain_character_varying AS character varying;
+  CREATE DOMAIN domain_date AS date;
+  CREATE DOMAIN domain_time AS time;
+  CREATE DOMAIN domain_timestamp AS timestamp;
+
+  CREATE TABLE domain_test (
+      id SERIAL PRIMARY KEY,
+      field_int domain_int,
+      field_smallint domain_smallint,
+      field_bigint domain_bigint,
+      field_numeric domain_numeric,
+      field_real domain_real,
+      field_double_precision domain_double_precision,
+      field_boolean domain_boolean,
+      field_text domain_text,
+      field_character domain_character,
+      field_character_varying domain_character_varying,
+      field_date domain_date,
+      field_time domain_time,
+      field_timestamp domain_timestamp
+  );
```

Comment:
Worth adding domain types on `json` and `jsonb`. Both these should be serialized as strings in GraphQL.

Reply:
Good catch, didn't even think about it.